samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -229,16 +229,12 @@ LPN configuration
 =================
 
 To make the light switch run as an LPN, set :makevar:`EXTRA_CONF_FILE` to :file:`overlay-lpn.conf` when building the sample.
-For example, when building from the command line, use the following command, where *board_target* is the target for the development kit for which you are building:
-
-.. parsed-literal::
-   :class: highlight
-
-   west build -b *board_target* -p -- -DEXTRA_CONF_FILE="overlay-lpn.conf"
 
 The configuration overlay :file:`overlay-lpn.conf` enables the LPN feature, and alters certain configuration options to further lower the power consumption.
 To review the specific alterations, open and inspect the :file:`overlay-lpn.conf` file.
-For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
+
+See :ref:`cmake_options` for instructions on how to add the :makevar:`EXTRA_CONF_FILE` option.
```

Comment:
I don't like these set of changes. They are making the documentation more convoluted. If I navigate to "NCS" page link, I don't find anywhere mention of EXTRA_CONF_FILE, and then I have to guess where to find this option and click on bunch of links. After few hit-and-miss clicks I finally discover this page: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16741/nrf/config_and_build/configuring_app/cmake/index.html#cmake-options which shows how to use EXTRA_CONF_FILE. Not a good UX, even for me.

For mesh samples, it is important that this EXTRA_CONF_FILE is stated and linked directly into the main sample documentation. It is the key part of the full sample evaluation. 

Therefore, I suggest to retain old text, and instead of ":ref:`zephyr:important-build-vars`" point it to [this](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-16741/nrf/config_and_build/configuring_app/cmake/index.html#cmake-options:~:text=DSB_CONF_FILE%3D%3Cfile_name%3E.conf-,EXTRA_CONF_FILE,-Provide%20additional%20Kconfig) location.

What do you think?

Reply:
I agree, @omkar3141 , and the page you are suggesting is exactly the page that we're linking to with ":ref:`cmake_options`".

Reply:
Is it possible to add a link for EXTRA_CONF_FILE text itself?

Reply:
It is possible, but it is not the style and formatting we follow in the docs.

Reply:
Ok. Then may I suggest to rethink about this later in your team and change it? Having in-place links to specific target landing paragraph or table row or picture, etc., is incredibly useful. See how KConfig options are presented in the docs ([example](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/libraries/bluetooth_services/mesh/gen_onoff_cli.html#:~:text=To%20make%20these,instance%20get%2Dall)). To me such CMAKE options also fall in the same category. It will be nice to enable such in-place links for convivence for end-users of SDK docs. This generally improves UX.

FYI @reidarma